torch/onnx/symbolic_opset10.py

```diff
@@ -69,46 +69,6 @@ def _apply(fn):
     return _apply
 
 
-@_onnx_symbolic("aten::div")
-@_beartype.beartype
-def div(g: jit_utils.GraphContext, self, other, *args):
-    if len(args) == 0:
-        return opset9.true_divide(g, self, other)
-    else:
-        return _div_rounding_mode(g, self, other, *args)
-
-
-@symbolic_helper.parse_args("v", "v", "s")
-@_beartype.beartype
-def _div_rounding_mode(g: jit_utils.GraphContext, self, other, rounding_mode):
-    if rounding_mode == "floor":
-        return _floor_divide(g, self, other)
-    else:
-        return opset9._div_rounding_mode(g, self, other, rounding_mode)
-
-
-@_onnx_symbolic("aten::_floor_divide")
-@_beartype.beartype
```

Comment:
Looks like what is in opset 10 is just a duplicate of opset 9. @BowenBao did I miss anything?